packages/domain-search/src/ui/domain-search-controls/filter-popover-label.tsx

```diff
@@ -0,0 +1,16 @@
+type Props = {
+	text: string;
+};
+
+export const FilterPopoverLabel = ( { text }: Props ) => {
+	return (
+		<div
+			className="domain-search-controls__filters-popover-list-label"
+			key={ text }
+			onKeyDown={ () => {} }
```

Comment:
What's the point of this `onKeyDown` handler? Also, why use a `div` and not a `span` for the label?

Reply:
This was a placeholder 😬 I'll change the `div` to `span`, it makes more sense